.github/workflows/auto-publish.yml

```
@@ -87,39 +91,104 @@ jobs:
+
+  release:
+    needs: [build, publish]
+    if: ${{ needs.build.outputs.dist_tag == 'latest' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      # 步骤10: 发布组件到GitHub Release
       - name: Release
-        if: ${{ steps.parse_tag.outputs.dist_tag == 'latest' }}
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)